src/components/Header.tsx

```diff
@@ -24,3 +24,27 @@ export default function Header() {
     </>
   );
 }
+
+type IndividualPageHeaderProps = {
+  title: string;
+};
+
+export function IndividualPageHeader({title}: IndividualPageHeaderProps) {
```

Comment:
why not keep the existing Header function, but now it would accept isHomepage (default to false), and then do
```
....
<Grid item xs={12} sm={6}>
{
isHomepage ?  
<HeadingSub>Network</HeadingSub>
  <Typography variant="h1" component="h1" gutterBottom>
            Aptos Governance
  </Typography>
:
<Typography variant="h3" component="h3">
            {title}
 </Typography>
}
</Grid>
....
```
can save confusion and check conditions in the page component level

Reply:
That way title would not be needed/used for the home page title which will be kinda confusing. I'd rather keep them as two component. Although good point with the readability. I'm changing the name "Header" to "HomePageHeader" to be more clear.